README.md

```diff
@@ -126,17 +125,33 @@ Then create a new env with `python3 -m venv <name of your venv>`
 
 Activate the venv with `source <name of your venv>/bin/activate`
 
+Install poetry with `pip install poetry`
+
+Install setuptools with `pip install setuptools`
```

Comment:
Why is `setuptools` needed? Doesn't `poetry` handle everything that setuptools did?

Reply:
Sadly setuptools is needed as pkg_resources is needed for pyfiglet. But I will remove it from the documentation. as it is not needed here. It will be installed with `poetry install`

Reply:
So it's just a transitive dependency from `pyfiglet` that we don't really rely on then?